src/app/tests/suites/commands/system/SystemCommands.cpp

```diff
@@ -53,7 +53,7 @@ CHIP_ERROR SystemCommands::Reboot(uint16_t discriminator)
     constexpr const char * scriptName = "Reboot.py";
 
     char command[128];
-    VerifyOrReturnError(snprintf(command, sizeof(command), "%s%s %u", scriptDir, scriptName, discriminator) >= 0,
+    VerifyOrReturnError(snprintf(command, sizeof(command), "%s%s --discriminator %u", scriptDir, scriptName, discriminator) >= 0,
```

Comment:
At the end of the day we should probably makes the 'discriminator' an optional argument but that's fine now.

Reply:
I'm actually in the middle of working on a PR to do just that :)